nautobot/extras/plugins/views.py

```diff
@@ -69,7 +69,7 @@ def _get_plugin_entry(plugin, app_config, request, format):
         api_app_name = f"{app_config.name}-api"
         try:
             entry = (
-                getattr(app_config, "base_url", app_config.label),
+                getattr(app_config, "base_url", app_config.name.replace("_", "-")),
```

Comment:
This seems like an incorrect change - looking at nautobot/extras/plugins/urls.py, `app_config.label` **is** what is used as a fallback if `app_config.base_url` is not set. Maybe something needs to be changed there as well?

Reply:
Ok I have figured out the fix, but not quite sure I like it. Replacing the `_` will slugify the value so the URL is valid. It still works if you click on the link, but doesn't reflect the correct URL.

Do you think we should maybe override the `__init__` within the `PluginConfig` and add the `base_url` attribute? Or create another attribute to default to besides the label?

Reply:
It's not clear to me why we need to replace `_` with `-` at all - can you elaborate? If the issue is that the `label` may not be valid for a URL, should we be using `slugify` for a full-fledged fixup instead of just replacing one character with another?

Reply:
Yes the `slugify` would work here as well.

This is more or less to keep it in line with the way the rest of the URLs, etc. are displayed and used, but we don't have to do any manipulation and just change the logic as you suggested.

If they don't provide a `base_url` then it would look like the following.

```json
{
    "installed-plugins": "http://myohman.local:8000/api/plugins/installed-plugins/",
    "eox_notices": "http://myohman.local:8000/api/plugins/eox_notices/"
}
```

While others will be slugified. Do we want to slugify the label or just keep it as is for now?

Reply:
I think we should just leave label as is because this would technically be a breaking change in non null instances.

Reply:
That works for me. I believe everything is in order for this PR now. Let me know if y'all have anything else.